src/contracts/CLSynchronicityPriceAdapterBaseToPeg.sol

```diff
@@ -78,6 +89,8 @@ contract CLSynchronicityPriceAdapterBaseToPeg is ICLSynchronicityPriceAdapter {
       return 0;
     }
 
-    return (assetToPegPrice * int256(10 ** DECIMALS)) / (baseToPegPrice);
+    return
+      (assetToPegPrice * int256(10 ** DECIMALS) * int256(10 ** BASE_TO_PEG_DECIMALS)) /
+      (baseToPegPrice * int256(10 ** ASSET_TO_PEG_DECIMALS));
```

Comment:
```suggestion
      (assetToPegPrice * int256(10 ** (DECIMALS + BASE_TO_PEG_DECIMALS - ASSET_TO_PEG_DECIMALS)) /
      baseToPegPrice;
```

If we reformat it that way, then it will be clear, that to keep certain precision we need to have extra require like:
`DECIMALS + BASE_TO_PEG_DECIMALS > X + ASSET_TO_PEG_DECIMALS`
Where **X** is the target minimal precision.

Overall, there is no point in keeping `ASSET_TO_PEG_DECIMALS` in the denominator because it will not improve precision for sure.

cc: @eboadom